pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -1010,6 +1019,44 @@ func getJmsConfig(environment api.EnvironmentType, jms *api.KieAppJmsObject) (*a
 	return jms, nil
 }
 
+func getKafkaConfig(kafka *api.KafkaExtObject) (*api.KafkaExtObject, error) {
+	if kafka == nil || !kafka.ExtEnabled {
+		return nil, nil
+	}
+
+	var override = false
+	defaultKafka := api.KafkaExtObject{}
+	//if something is missing we set mandatory defaults
+	if kafka.MaxBlockMs == 0 {
+		defaultKafka.MaxBlockMs = 2000
+		override = true
```

Comment:
instead of using defaultKafka and override, why not just set `kafka.MaxBlockMs = 2000` directly? you could do the same for the rest of the defaults and then get rid of the mergo logic. is there something i'm missing?

Reply:
Ok I was thinking that mergo was the pattern to set defaults